js/apps/admin-ui/src/components/role-mapping/RoleMapping.tsx

```diff
@@ -250,8 +250,8 @@ export const RoleMapping = ({
         ]}
         emptyState={
           <ListEmptyState
-            message={t("noRoles")}
-            instructions={t("noRolesInstructions")}
+            message={t("noRoles-" + type)}
+            instructions={t("noRolesInstructions-" + type)}
```

Comment:
We can use a template string here instead.
```suggestion
            message={t(`noRoles-${type}`)}
            instructions={t(`noRolesInstructions-${type}`)}
```